packages/teams-js/src/public/dialog.ts

```diff
@@ -179,7 +159,7 @@ export namespace dialog {
         throw errorNotSupportedOnPlatform;
       }
 
-      sendMessageToParent('messageForParent', [message]);
+      sendMessageToParentWithVersion('v1', 'messageForParent', [message]);
```

Comment:
is this v1?

Reply:
Yes, it doesn't have any other function (which we call it deprecated) performs the same task